README.rst

```diff
@@ -17,19 +17,18 @@ Download
 
 Stable release
 ~~~~~~~~~~~~~~
-Binaries and packages of the latest stable release are available at `https://github.com/mozilla/sops/releases <https://github.com/mozilla/sops/releases>`_.
+Binaries and packages of the latest stable release are available at `https://github.com/getsops/sops/releases <https://github.com/getsops/sops/releases>`_.
 
 Development branch
 ~~~~~~~~~~~~~~~~~~
-For the adventurous, unstable features are available in the `develop` branch, which you can install from source:
+For the adventurous, unstable features are available in the `main` branch, which you can install from source:
 
 .. code:: bash
 
 	$ mkdir -p $GOPATH/src/go.mozilla.org/sops/
-        $ git clone https://github.com/mozilla/sops.git $GOPATH/src/go.mozilla.org/sops/
-        $ cd $GOPATH/src/go.mozilla.org/sops/
-        $ git checkout develop
-        $ make install
+  $ git clone https://github.com/getsops/sops.git $GOPATH/src/go.mozilla.org/sops/
+  $ cd $GOPATH/src/go.mozilla.org/sops/
+  $ make install
```

Comment:
Minor formatting issue: `$ mkdir -p $GOPATH/src/go.mozilla.org/sops/` should be aligned with the new new commands